samples/star-wars-api/HttpHandlers.fs

```diff
@@ -38,22 +42,28 @@ module HttpHandlers =
     let private graphQL (next : HttpFunc) (ctx : HttpContext) =
         task {
 
+            let logger = ctx.RequestServices.CreateLogger moduleType
+
             // TODO: validate the result
             let toResponse documentId =
                 function
                 | Direct   (data, errs) ->
+                    if not <| logger.IsEnabled LogLevel.Trace then
+                        logger.LogTrace("Produced direct GraphQL response with documentId = '{documentId}' and data = {data}", documentId, JsonSerializer.Serialize(data, Json.serializerOptions))
                     { DocumentId = documentId
                       Data = data
                       Errors = errs }
                 | Deferred (data, errs, deferred) ->
-                    // TODO: Print to logger
-                    deferred |> Observable.add (fun d -> printfn "Deferred: %s" (JsonSerializer.Serialize(d, Json.serializerOptions)))
+                    if not <| logger.IsEnabled LogLevel.Trace then
+                        logger.LogTrace("Produced deferred GraphQL response with documentId = '{documentId}' and data = {data} and deferred = {deferred}", documentId, data, deferred)
```

Comment:
`deferred` is `IObservable` as I remember. We don't need it as nothing interesting there